### DISCUSSION TITLE
Can we assign screen overlay or filter to specific console

### DISCUSSION THREAD

**Discussion by F00FIGHTRS** [OnionUI/Onion#744] [Q&A] [answered]
I love the variety of screen overlays and filters; however, if I close a game and reopen it- then I have to reapply them again. Is there a way to apply overlays/filters to a specific console and have it saved? Is there a way to apply overlays/filters to a specific ROM and have it saved?

**Comment by Aemiii91** [comment, marked as answer]
Yes, you can save specific overrides. See this: https://github.com/OnionUI/Onion/wiki/Frequently-Asked-Questions-%28FAQ%29#how-do-i-save-retroarch-settings

**Reply by F00FIGHTRS** [reply]
OMG! Thx so much!
